src/main/java/com/epam/petclinic/order/config/ResourceServerConfiguration.java

```diff
@@ -1,30 +1,35 @@
 package com.epam.petclinic.order.config;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 import org.springframework.security.oauth2.config.annotation.web.configurers.ResourceServerSecurityConfigurer;
-import org.springframework.security.oauth2.provider.token.TokenStore;
 
+/**
+ * Configures access to service resources.
+ *
+ * Date: 5/19/2017
+ *
+ * @author Andrei Astaschenko
+ */
 @Configuration
 @EnableResourceServer
 public class ResourceServerConfiguration extends ResourceServerConfigurerAdapter {
-
-    @Autowired
-    private TokenStore tokenStore;
-
     @Override
     public void configure(HttpSecurity http) throws Exception {
         http
             .authorizeRequests()
-                // example of adding request security
-                //.antMatchers(HttpMethod.GET, "/orders").hasAuthority("user")
-                .anyRequest().authenticated();
+            // example of adding request security
+            .antMatchers(HttpMethod.GET, "/orders").authenticated()
+            .and().authorizeRequests()
+            .antMatchers(HttpMethod.GET, "/orders/clinic-test").hasAuthority("admin")
+            .anyRequest().authenticated();
     }
+
     @Override
     public void configure(ResourceServerSecurityConfigurer resources) {
-        resources.resourceId("ORDER-SERVICE").tokenStore(tokenStore);
```

Comment:
Solved, specified public key is used to decode token comiing from Authentication server (there are principle, permissions and other importnat information to reduce number of calls to AUTH-SERVER). Add comment to key in property file